templates/email/ChangePasswordEmail.ss

```diff
@@ -1,11 +1,11 @@
-<p><% _t('ChangePasswordEmail_ss.HELLO', 'Hi') %> $FirstName,</p>
+<p><%t ChangePasswordEmail_ss.HELLO 'Hi' %> $FirstName,</p>
 
 <p>
-	<% _t('ChangePasswordEmail_ss.CHANGEPASSWORDTEXT1', 'You changed your password for', 'for a url') %> $AbsoluteBaseURL.<br />
-	<% _t('ChangePasswordEmail_ss.CHANGEPASSWORDTEXT2', 'You can now use the following credentials to log in:') %>
+	<%t ChangePasswordEmail_ss.CHANGEPASSWORDTEXT1 'You changed your password for' is 'for a url' %> $AbsoluteBaseURL.<br />

```

Comment:
`is 'context goes here'` is the way to pass context information through the new translation syntax. As far as I can tell, the context is never actually used by anything at all, but I figured it best to leave it in as a precaution